elfi/methods/parameter_inference.py

```diff
@@ -1287,3 +1287,202 @@ def sample(self,
             threshold=float(posterior.threshold),
             n_sim=self.state['n_sim'],
             seed=self.seed)
+
+
+class ExponentialBOLFI(BayesianOptimization):
```

Comment:
Better to inherit BOLFI to avoid duplicate code. Then just override the relevant methods e.g. by running the method from super, capture its output and return the modified result.